projects/js-packages/components/components/status/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { __ } from '@wordpress/i18n';
+import classNames from 'classnames';
+import styles from './style.module.scss';
+
+interface StatusProps {
+	status?: 'active' | 'error' | 'inactive' | 'action' | 'initializing';
+	label?: string;
+	className?: string;
+}
+
+const Status = ( { className, label, status = 'inactive', ...rest }: StatusProps ): JSX.Element => {
+	const defaultLabels: Record< string, string > = {
+		active: __( 'Active', 'jetpack' ),
+		error: __( 'Error', 'jetpack' ),
+		action: __( 'Action needed', 'jetpack' ),
+		inactive: __( 'Inactive', 'jetpack' ),
+		initializing: __( 'Setting up', 'jetpack' ),
+	};
+
+	return (
+		<span
+			className={ classNames(
+				styles.status,
+				{
+					[ styles[ `status--${ status }` ] ]: status,
+				},
+				className
+			) }
+			{ ...rest }
+		>
+			<span className={ styles.status__indicator } />
+			{ label || label === '' ? (
+				<span className={ styles.status__label }>{ label }</span>
+			) : (
+				<span className={ styles.status__label }>{ defaultLabels[ status ] }</span>
+			) }
```

Comment:
minor enhancement. Feel free to ignore it.
```suggestion
				<span className={ styles.status__label }>
					{ label || label === '' ? label : defaultLabels[ status ] }
				</span>
```

Reply:
> Feel free to ignore it.

I will do no such thing! Applied your suggestion here: https://github.com/Automattic/jetpack/pull/30209/commits/0b7c845d0def850fe5099d654fe2371ad25a17b2